hbase-server/src/main/java/org/apache/hadoop/hbase/coprocessor/CoprocessorHost.java

```diff
@@ -143,8 +143,16 @@ protected void loadSystemCoprocessors(Configuration conf, String confKey) {
     if (defaultCPClasses == null || defaultCPClasses.length == 0)
       return;
 
-    int priority = Coprocessor.PRIORITY_SYSTEM;
+    int currentSystemPriority = Coprocessor.PRIORITY_SYSTEM;
     for (String className : defaultCPClasses) {
+      String[] classNameAndPriority = className.split("\\|");
```

Comment:
May be good to update doc for `hbase.coprocessor.regionserver.classes` in section `Restricting Coprocessor Usage` of cp.adoc?

Reply:
Sure, I added some notes to cp.adoc and updated the PR, @virajjasani .